features/networking/egress-ip.feature

```diff
@@ -136,18 +135,18 @@ Feature: Egress IP related features
     # Patch egress IP to the project twice
     Given I have a project
     And as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
-      | {"egressIPs": ["<%= cb.newip %>"]} |
+      | {"egressIPs": ["<%= cb.valid_ips[0]%>"]} |
     And as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
-      | {"egressIPs": ["<%= cb.valid_ip %>"]} |
+      | {"egressIPs": ["<%= cb.valid_ips[1] %>"]} |
 
     # Check the egress ip is the last one applied
     And I wait up to 30 seconds for the steps to pass:
     """
     When I run command on the "<%= node.name%>" node's sdn pod:
       | bash | -c | ip address show <%= cb.interface %> |
     Then the step should succeed
-    And the output should contain "<%= cb.valid_ip %>"
-    And the output should not contain "<%= cb.newip %>"
+    And the output should contain "<%= cb.valid_ips[1] %>"
+    And the output should not contain "<%= cb.valid_ips[0] %>"
     """
```

Comment:
@huiran0826 To make sure the last one is applied, I would suggest to check both valid_ip[0] and valid_ip[1]:

First step apply {"egressIPs": ["<%= cb.valid_ips[0]%>"]} , then cb.valid_ips[0] will be shown in pod,

Second step apply {"egressIPs": ["<%= cb.valid_ip[1] %>"]}, then cb.valid_ips[1] will be shown in pod

Reply:
@weliang1 Thanks, updated it.